packages.json

```diff
@@ -5618,5 +5618,18 @@
     "description": "Simple extensible implementation of Porter stemmer algorithm",
     "license": "MIT",
     "web": "https://github.com/iourinski/porter"
+  },
+  {
+    "name": "arrayfire-nim",
```

Comment:
Remove the dash from the name please. This needs to match the name of your .nimble file which is ``ArrayFireNim`` (the case doesn't matter though).